server/imports/api/env.os_families.js

```diff
@@ -0,0 +1,52 @@
+//secure keys are not defined, but may be implemented
+
+/** supported linux families for lab virtual machines
+ * although instructors can pass system objects like the ones below,
+ * no other os has been tested and may not function as expected
+ * if the os_family field is not provided, all systems default to alpine
+ */
+var os_families = {
+  //alpine defaults to the tuxlab_labvm image
+  alpine: {
+    username: "root",
+    password: "",
+    key: "",
+    os_family: "alpine",
+    image: "tuxlab_labvm",
+    ssh_port: 22
+  },
+
+  redhat: {
+    username: "",
+    password: "",
+    key: "",
+    os_family: "redhat",
+    image: ""
+  },
+
+  ubuntu: {
+    username: "",
+    password: "",
+    key: "",
+    os_family: "ubuntu",
+    image: "ubuntu"
```

Comment:
just delete "ubuntu" from the images section- it wouldn't work because SSH isn't enabled